### PR TITLE
Update email

### PR DESCRIPTION
**Pull request by vdobrotskyi-varonis**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/VaronisDataSecurityPlatform/pack_metadata.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "currentVersion": "1.0.1",
     "author": "Varonis",
     "url": "",
-    "email": "dcohen@varonis.com",
+    "email": "https://www.varonis.com/support",
```
Comment (yaakovpraisler) [inline Packs/VaronisDataSecurityPlatform/pack_metadata.json:8]: Please add this url to the `url` field and add here your email

**Review by yaakovpraisler — CHANGES_REQUESTED** [review]
Hi @vdobrotskyi-varonis - Please fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) so we'll can continue

File: Packs/VaronisDataSecurityPlatform/pack_metadata.json (hunk 1 above)
Comment (vdobrotskyi-varonis) [reply Packs/VaronisDataSecurityPlatform/pack_metadata.json:8]: @yaakovpraisler can we have only url and skip email?

Comment (yaakovpraisler) [reply Packs/VaronisDataSecurityPlatform/pack_metadata.json:8]: Yes, but please move the url to the `url` field

Comment (vdobrotskyi-varonis) [reply Packs/VaronisDataSecurityPlatform/pack_metadata.json]: @yaakovpraisler sure, done. Also I filled contribution form

**Review by yaakovpraisler — APPROVED** [review]

**Pull request merged**